cmd/cephcsi.go

```diff
@@ -123,13 +123,15 @@ func main() {
 	util.DefaultLog("Driver version: %s and Git version: %s", util.DriverVersion, util.GitCommit)
 
 	if conf.Vtype == "" {
-		klog.Fatalln("driver type not specified")
+		klog.Errorln("driver type not specified")
+		os.Exit(1)
```

Comment:
Maybe add a short helper function, something like `func die(msg string)` (or whatnot) to avoid some boilerplate around remembering to call exit and it could make future changes simpler?

Reply:
Yeah, functionalizing graceful exit and calling with an arg is preferred.

Reply:
Done!